.github/workflows/ci.yml

```diff
@@ -9,13 +9,16 @@ name: ci
 
 jobs:
   lint-unit:
-    uses: sous-chefs/.github/.github/workflows/lint-unit.yml@2.0.4
+    # uses: sous-chefs/.github/.github/workflows/lint-unit.yml@2.0.4
+    uses: b-dean/sous-chefs.github/.github/workflows/lint-unit.yml@chef-version
```

Comment:
Could you roll the changes from here into a PR for the .github repo please? We keep the workflow files standard across all cookbooks and it's a helpful addition.

Reply:
@bmhughes it doesn't have a tagged version over there, I updated it to the revision from `main`, if you tag it I can amend and force push again.